### PR TITLE
Support CosmosDb real database types

### PR DESCRIPTION
**Pull request by chagong**
Support CosmosDb real database types

### CODE REVIEW

File: azext_connect/custom.py [hunk 1]
```diff
@@ -382,10 +383,49 @@ def create_resource(service, resource_group, deployment_id, settings, para_dict)
...
+            if item.Name == COSMOS_CAPABILITES[0]:
+                return COSMOS_DATABASE_TYPE[0]
+            if item.Name == COSMOS_CAPABILITES[1]:
+                return COSMOS_DATABASE_TYPE[1]
+            if item.Name == COSMOS_CAPABILITES[2]:
+                return COSMOS_DATABASE_TYPE[2]
+            return COSMOS_DATABASE_TYPE[3]
+    if kind == COSMOSDB_KIND[1]:
+        return COSMOS_DATABASE_TYPE[4]
+    return 'unknown'
```
Comment (dooriya) [inline azext_connect/custom.py]: Seems we don't handle the case of "unknown" database type in the caller

File: azext_connect/custom.py [hunk 2]
```diff
@@ -382,10 +383,49 @@ def create_resource(service, resource_group, deployment_id, settings, para_dict)
         spring_cloud_handler(resource_group, deployment_id, settings, para_dict)
 
 
+COSMOSDB_KIND = ['GlobalDocumentDB', 'MongoDB', 'Parse']
```
Comment (dooriya) [inline azext_connect/custom.py:386]: Shall we support this? What't the CLI experience if the input cosmos db is kind of "Parse"?

Comment (chagong) [reply azext_connect/custom.py:386]: Looks like "Parse" is not currently supported by Azure CosmosDB. But "Parse" is listed in the rest API docs.

File: azext_connect/custom.py (hunk 1 above)
Comment (chagong) [reply azext_connect/custom.py]: I changed to throw exception here.

**Review by KennethBWSong — APPROVED** [review]

**Pull request merged**